app/Http/Requests/AddResponseToTaskRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Rules\CheckEncryptedInput;
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddResponseToTaskRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'content' => 'required',
+            'task_id' => [
+                'required',
+                'string',
+                new CheckEncryptedInput(),
```

Comment:
Laravel has validation rule to check old password `current_password`. There's no need for custom validation. Docs link https://laravel.com/docs/8.x/validation#rule-current-password

Reply:
I'm not validating password here, I pretended to "hide" real task_id in encrypted hidden input for security reason and avoid to post response just changing url "/task/123/add-response" for example. In fact this doesn't work as I expected and I have to completely review this logic.